runtime/runtime/src/ext.rs

```diff
@@ -379,11 +379,12 @@ impl<'a> Contract for RuntimeContractExt<'a> {
         let code_hash = self.hash();
         let version = self.current_protocol_version;
         let chain_id = self.chain_id;
+        // We are removing the check to match code_hash_matches_wallet_contract
```

Comment:
nit. once you remove the code below, this comment will not refer to any check after merging.